src/main/java/com/github/black0nion/blackonionbot/commands/moderation/ClearCommand.java

```diff
@@ -1,47 +1,45 @@
 package com.github.black0nion.blackonionbot.commands.moderation;
 
-import java.time.OffsetDateTime;
-import java.time.ZoneOffset;
-import java.util.ArrayList;
-import java.util.List;
-
 import com.github.black0nion.blackonionbot.commands.SlashCommand;
 import com.github.black0nion.blackonionbot.commands.SlashCommandEvent;
+import com.github.black0nion.blackonionbot.utils.Placeholder;
 import com.github.black0nion.blackonionbot.wrappers.jda.BlackGuild;
 import com.github.black0nion.blackonionbot.wrappers.jda.BlackMember;
 import com.github.black0nion.blackonionbot.wrappers.jda.BlackUser;
-import com.github.black0nion.blackonionbot.utils.Placeholder;
-
 import net.dv8tion.jda.api.Permission;
 import net.dv8tion.jda.api.entities.Message;
 import net.dv8tion.jda.api.entities.TextChannel;
 import net.dv8tion.jda.api.events.interaction.command.SlashCommandInteractionEvent;
 import net.dv8tion.jda.api.interactions.commands.OptionMapping;
 import net.dv8tion.jda.api.interactions.commands.OptionType;
 import net.dv8tion.jda.api.interactions.commands.build.Commands;
+import net.dv8tion.jda.api.interactions.commands.build.OptionData;
+import org.jetbrains.annotations.NotNull;
+
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+import java.util.ArrayList;
+import java.util.List;
 
 public class ClearCommand extends SlashCommand {
 
 	public ClearCommand() {
 		super(builder(Commands.slash("clear", "Clear a certain amount of messages")
-			.addOption(OptionType.INTEGER, "amount", "Amount of messages to delete", true))
-			.setRequiredPermissions(Permission.MESSAGE_MANAGE)
-			.setEphemeral(true)
-			.setRequiredBotPermissions(Permission.MESSAGE_MANAGE));
+				.addOptions(new OptionData(OptionType.INTEGER, "amount", "Amount of messages to delete", true)
+						.setRequiredRange(2, 100))).setRequiredPermissions(Permission.MESSAGE_MANAGE).setEphemeral(true)
+								.setRequiredBotPermissions(Permission.MESSAGE_MANAGE));
 	}
 
 	@Override
-	public void execute(SlashCommandEvent cmde, SlashCommandInteractionEvent e, BlackMember member, BlackUser author, BlackGuild guild, TextChannel channel) {
+	public void execute(@NotNull SlashCommandEvent cmde, @NotNull SlashCommandInteractionEvent e, BlackMember member,
+			BlackUser author, BlackGuild guild, @NotNull TextChannel channel) {
 		try {
 			final Integer amount = e.getOption("amount", OptionMapping::getAsInt);
-			if (amount == null || amount < 2 || amount > 100) {
-				cmde.send("toomanymessages" );
-				return;
-			}
 
 			try {
 				channel.getIterableHistory().cache(false).queue(msgs -> {
-					final OffsetDateTime firstValidDate = OffsetDateTime.now(ZoneOffset.UTC).minusWeeks(2).plusSeconds(1);
+					final OffsetDateTime firstValidDate = OffsetDateTime.now(ZoneOffset.UTC).minusWeeks(2)
+							.plusSeconds(1);
```

Comment:
Please make a newline for either both minusWeeks and plusSeconds or dont use a newline